IntegrationTests/SPMIntegration/SPMIntegration.xcodeproj/project.pbxproj

```diff
@@ -559,7 +558,7 @@
 			isa = XCRemoteSwiftPackageReference;
 			repositoryURL = "https://github.com/RevenueCat/purchases-ios.git";
 			requirement = {
-				branch = main;
+				branch = "release/4.0.0-beta.2";
```

Comment:
I'll switch these SPM changes back when we merge the changes to main after beta 2 release.

Reply:
I don't think this is needed, since we update it programmatically in `config.yml` when running integration tests, deployment checks and releases: 
https://github.com/RevenueCat/purchases-ios/blob/main/.circleci/config.yml#L95

Reply:
Right. Removed!